dotnet/src/dotnetframework/GxClasses/Core/GXApplication.cs

```diff
@@ -2199,7 +2199,7 @@ public string GetCompleteURL(string file)
 		{
 			String fout = file.Trim();
 
-			if (PathUtil.IsAbsoluteUrl(file) || file.StartsWith("//") || (file.Length > 2 && file[1] == ':'))
```

Comment:
file.StartsWith("//") seems to be the case for "scheme relative" or "protocol relative" URIs. (ex: //cdn.example.com/js_file.js)

Now the method IsAbsoluteUrlOrAnyScheme() checks for only one "/": "url.StartsWith("/")"
So, now it's also matching "Host relative URIs" (ex: /static/myfile.js"). 

Maybe it's ok, just wanted to let know.

Reply:
Agree. Fixed mistake.